src/classes/BookingsLog.js

```diff
@@ -1,7 +1,56 @@
 class BookingsLog {
-  constructor() {
+  constructor(bookings) {
+    this.bookings = bookings;
+  }
+
+  getCustomerBookings(customer) {
+    return this.bookings.filter(booking => customer.id === booking.userID)
+  }
 
+  calculateTotalSpent(roomListings, customer) {
+    const customerBookings = this.getCustomerBookings(customer);
+    return roomListings.hotelRooms.reduce((acc, listing) => {
```

Comment:
Think about what you're returning with the `acc` and give it a more specific name.